modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -21,6 +21,7 @@ The following table describes the CSI drivers that are installed with {product-t
 |Google Cloud Platform (GCP) persistent disk (PD) (Tech Preview)| ✅ | - | ✅
 |Microsoft Azure Disk (Tech Preview) | ✅ | ✅ | ✅
 |Microsoft Azure Stack Hub | ✅ | ✅ | ✅
+|Microsoft Azure File (Tech Preview) | ✅ | - | ✅
```

Comment:
@bertinatto Hi, as we discussed before, Azure file only support create volumesnapshot but not support restore, should we declare support "CSI volume snapshots" here?

Reply:
Good catch, @duanwei33. Apparently Azure File misses an API to restore snapshots, so the CSI driver can't fully support snapshots for now. I think we should reflect that in the docs.